grafana/provisioning/dashboards/businessInteligence.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: 1
+providers:
+- name: 'businessInteligence'
+  orgId: 1
+  folder: 'Business Inteligence'
+  type: file
+  disableDeletion: true
+  updateIntervalSeconds: 300
+  options:
+    path: /etc/grafana/provisioning/dashboards/businessInteligence
+
```

Comment:
This yaml will create the `Business Inteligence` folder.
All the json files included at the folder `/etc/grafana/provisioning/dashboards/businessInteligence` will be provisioned automatically at the `Business Inteligence` folder.

So if you want to add new dashboards, you just gotta add their corresponding JSONs files at this directory.